src/ui/Startup.cs

```diff
@@ -263,6 +263,7 @@ public void ConfigureServices(IServiceCollection services)
             });
             services.AddScoped<SearchAndCompare.UI.Shared.Features.IFeatureFlags, SearchAndCompare.UI.Shared.Features.FeatureFlags>();
             services.AddSingleton<ISearchAndCompareUrlService>(x => new SearchAndCompareUrlService(Configuration.GetValue("SearchAndCompare:UiBaseUrl", "")));
+            services.AddSingleton<IFrontendUrlService>(x => new FrontendUrlService(Configuration));
```

Comment:
services.AddSingleton<IFrontendUrlService, FrontendUrlService>();

Don't think you need to new it up